.gitignore

```diff
@@ -10,7 +10,8 @@
 
 erl_crash.dump
 
-/priv/static
+/priv/static/assets/
```

Comment:
Align the `static` folder with a `gitignore` file from the up-to-date Phoenix template